packages/network-console-shared/src/environments/native/native-env-format.ts

```diff
@@ -56,6 +58,8 @@ export function serializeNativeEnvironment(
     settings: INetConsoleParameter[],
     addTabs = false,
 ): string {
+    deprecated('serializeNativeEnvironment');
```

Comment:
what is this for?

Reply:
All of the old collections and environments code and interfaces are being deprecated and will be removed in 0.10. I added this in the parent branch to emit console warnings if the code is being called.